### PR TITLE
Component: Button (size, style), header, logo, icon

### PR DESCRIPTION
**Pull request by DatQTruongCr**
Cảm ơn bạn nhiều, 
Nào bạn chỉ lại giúp mình với, nay mình viết tổ chức class tệ quá

### CODE REVIEW

File: src/scss/components/_brand.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+%flexItemCenter {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+@mixin painSquare($size) {
```
Comment (HuanDoan) [inline src/scss/components/_brand.scss:7]: I think it should be `paintSquare` !?

**Review by HuanDoan — COMMENTED** [review]
I left some comments in your code, pls check and fix it.

Besides, please note that the space next to the comma, colon, as well as `{`, `}`

File: src/button.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        th,
+        td {
+            border: 1px solid black !important;
+            border-collapse: collapse;
+        }
+
+        th,
+        td {
+            padding: 15px 15px !important;
+        }
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/button.html:30]: I will check again and fix all, after 1 hour again. Thank you your time for review my code

[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/button.html]: I fixed this problem. and pull request again.

File: src/scss/components/_button.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
+@mixin BtnDesign-color($Color,$colorText, $boderColor) {
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/scss/components/_button.scss]: I fixed this problem. and pull request again.

File: src/scss/components/_button.scss [hunk 4]
```diff
@@ -0,0 +1,82 @@
+@mixin BtnDesign-color($Color,$colorText, $boderColor) {
+    background-color: $Color;
+    color:$colorText;
+    border: 1px solid $boderColor;
+}
+
+@mixin BtnDesign-Size($padTBot, $padLR){
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/scss/components/_button.scss]: I fixed this problem. and pull request again.

File: src/scss/components/_button.scss [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+}
+.btn-small-l {
+    @include BtnDesign-Size(20px,53px)
+}
+.btn-small-m-2{
+    @include BtnDesign-Size(20px,41px)
+}
+.btn-small-m-1 {
+    @include BtnDesign-Size(13px,33px)
+}
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/scss/components/_button.scss]: I fixed this problem. and pull request again.

File: src/scss/components/_header.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+
+.header__navCta{
+  width: 40%;
+  margin-left: 90px;
+.header-nav{
+    display: flex;
+    justify-content: space-between;
+}
+}
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/scss/components/_header.scss]: I fixed this problem. and pull request again.

File: src/button.html [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+            <td>
+                <div class="btnBasic btn-yellow-bgr btn-large-l">Continue to Checkout</div>
+            </td>
+        </tr>
+
+
+
+    </table>
+
+
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/button.html]: I fixed this problem. and pull request again.

File: src/button.html [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+        <tr>
+            <td>btnBasic btn-yellow-bgr btn-large-l</td>
+            <td>
+                <div class="btnBasic btn-yellow-bgr btn-large-l">Continue to Checkout</div>
+            </td>
+        </tr>
+
+
+
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/button.html]: I fixed this problem. and pull request again.

File: src/scss/components/_brand.scss [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+@mixin painSquare($size) {
+    @extend %flexItemCenter;
+    width: $size;
+    height: $size;
+    min-width: 30px;
+    min-height: 30px;
+}
+
+.brand__containerSquare {
```
[the remark this answers is not in the document]
Comment (DatQTruongCr) [reply src/scss/components/_brand.scss]: I fixed this problem. and pull request again.

**Comment by HuanDoan** [comment]
@DatQTruongCr hey guy, pls remove all the assets & files related to the 404 page in this PR

**Pull request merged**